api/tests/unit/organisations/test_unit_organisations_tasks.py

```diff
@@ -136,3 +202,23 @@ def test_finish_subscription_cancellation(db: None):
     assert organisation2.num_seats == 1
     assert organisation3.num_seats == 1
     assert organisation4.num_seats == organisation_user_count
+
+
+def test_send_org_subscription_cancelled_alert(db: None, mocker: MockerFixture) -> None:
+    # Given
+    send_mail_mock = mocker.patch("users.models.send_mail")
+
+    # When
+    send_org_subscription_cancelled_alert(
+        organisation_name="Nike",
```

Comment:
Nike here makes me feel a bit uncomfortable for some reason?

Reply:
Agree with this.

Reply:
I forget we're developing Open Source and Nike could ruffle feathers. In general, I like realistic named tests, but for this one it's the right call to change it something neutral.